22:29:46.762 ERROR [File:ProfileService.java] [Func:create] [Line:35] [Message:org.springframework.dao.TransientDataAccessResourceException: PreparedStatementCallback; SQL []; SQL String cannot be empty; nested exception is java.sql.SQLException: SQL String cannot be empty]- org.springframework.dao.TransientDataAccessResourceException: PreparedStatementCallback; SQL []; SQL String cannot be empty; nested exception is java.sql.SQLException: SQL String cannot be empty
22:31:37.843 ERROR [File:ProfileService.java] [Func:create] [Line:35] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, ageGrade) VALUES (?,?)]; Field 'language' doesn't have a default value; nested exception is java.sql.SQLException: Field 'language' doesn't have a default value]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, ageGrade) VALUES (?,?)]; Field 'language' doesn't have a default value; nested exception is java.sql.SQLException: Field 'language' doesn't have a default value
22:48:39.599 ERROR [File:ProfileService.java] [Func:create] [Line:35] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, photoUrl, ageGrade) VALUES (?,?,?)]; Column 'photoUrl' cannot be null; nested exception is java.sql.SQLIntegrityConstraintViolationException: Column 'photoUrl' cannot be null]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, photoUrl, ageGrade) VALUES (?,?,?)]; Column 'photoUrl' cannot be null; nested exception is java.sql.SQLIntegrityConstraintViolationException: Column 'photoUrl' cannot be null
22:51:43.352 ERROR [File:ProfileService.java] [Func:create] [Line:35] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, photoUrl, ageGrade) VALUES (?,?,?)]; Field 'AccountIdx' doesn't have a default value; nested exception is java.sql.SQLException: Field 'AccountIdx' doesn't have a default value]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Profile (name, photoUrl, ageGrade) VALUES (?,?,?)]; Field 'AccountIdx' doesn't have a default value; nested exception is java.sql.SQLException: Field 'AccountIdx' doesn't have a default value
21:18:22.366 ERROR [File:AccountController.java] [Func:createAccount] [Line:89] [Message:com.example.demo.config.BaseException]- com.example.demo.config.BaseException
21:18:26.217 ERROR [File:AccountService.java] [Func:createAccount] [Line:53] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (password, email) VALUES (?,?)]; Field 'phoneNumber' doesn't have a default value; nested exception is java.sql.SQLException: Field 'phoneNumber' doesn't have a default value]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (password, email) VALUES (?,?)]; Field 'phoneNumber' doesn't have a default value; nested exception is java.sql.SQLException: Field 'phoneNumber' doesn't have a default value
21:18:26.219 ERROR [File:AccountController.java] [Func:createAccount] [Line:89] [Message:com.example.demo.config.BaseException]- com.example.demo.config.BaseException
21:20:20.365 ERROR [File:AccountService.java] [Func:createAccount] [Line:53] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (password, email) VALUES (?,?)]; Field 'phoneNumber' doesn't have a default value; nested exception is java.sql.SQLException: Field 'phoneNumber' doesn't have a default value]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (password, email) VALUES (?,?)]; Field 'phoneNumber' doesn't have a default value; nested exception is java.sql.SQLException: Field 'phoneNumber' doesn't have a default value
21:20:20.365 ERROR [File:AccountController.java] [Func:createAccount] [Line:89] [Message:com.example.demo.config.BaseException]- com.example.demo.config.BaseException
20:45:40.605 ERROR [File:ProfilePhotoProvider.java] [Func:getProfilePhoto] [Line:35] [Message:org.springframework.dao.EmptyResultDataAccessException: Incorrect result size: expected 1, actual 0]- org.springframework.dao.EmptyResultDataAccessException: Incorrect result size: expected 1, actual 0
20:48:13.183 ERROR [File:ProfilePhotoProvider.java] [Func:getProfilePhoto] [Line:37] [Message:org.springframework.dao.EmptyResultDataAccessException: Incorrect result size: expected 1, actual 0]- org.springframework.dao.EmptyResultDataAccessException: Incorrect result size: expected 1, actual 0
23:18:00.578 ERROR [File:ProfileController.java] [Func:manageProfile] [Line:53] [Message:com.example.demo.config.BaseException]- com.example.demo.config.BaseException
19:19:27.824 ERROR [File:NaverAccountController.java] [Func:getNaverAccountInfo] [Line:174] [Message:org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]]- org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]
19:32:22.232 ERROR [File:NaverAccountService.java] [Func:createNaverAccount] [Line:41] [Message:org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (email, password, socialLoginIdx, socialLoginType) VALUES (?,?,?,?)]; Data truncation: Data too long for column 'socialLoginIdx' at row 1; nested exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'socialLoginIdx' at row 1]- org.springframework.dao.DataIntegrityViolationException: PreparedStatementCallback; SQL [insert into Account (email, password, socialLoginIdx, socialLoginType) VALUES (?,?,?,?)]; Data truncation: Data too long for column 'socialLoginIdx' at row 1; nested exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'socialLoginIdx' at row 1
19:43:54.013 ERROR [File:NaverAccountController.java] [Func:getNaverAccountInfo] [Line:173] [Message:org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]]- org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]
20:17:29.706 ERROR [File:NaverAccountController.java] [Func:getNaverAccountInfo] [Line:174] [Message:org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]]- org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]
20:23:37.778 ERROR [File:NaverAccountController.java] [Func:getNaverAccountInfo] [Line:174] [Message:org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]]- org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]
20:31:05.435 ERROR [File:NaverAccountController.java] [Func:getNaverAccountInfo] [Line:174] [Message:org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]]- org.springframework.web.client.HttpClientErrorException$Unauthorized: 401 Unauthorized: [{"resultcode":"024","message":"Authentication failed (\uc778\uc99d \uc2e4\ud328\ud558\uc600\uc2b5\ub2c8\ub2e4.)"}]
